src/components/utils/IntegrationLogo.tsx

```diff
@@ -0,0 +1,39 @@
+import { Box } from "@mui/material";
+import Image from "next/image";
+
+import { IntegrationProfile } from "@/types/integration";
+
+function IntegrationLogo({ integrationProfile }: { integrationProfile: IntegrationProfile }) {
+    return (
+        <Box
+            sx={{
+                position: "relative",
+                height: 25,
+                width: 75,
+            }}
+        >
+            <Image
+                style={{
+                    filter: `
+                        drop-shadow(0px 0px 0 #fff)
+                        drop-shadow(1px 0px 0 #fff)
+                        drop-shadow(0px 1px 0 #fff)
+                        drop-shadow(-1px 0px 0 #fff)
+                        drop-shadow(0px -1px 0 #fff)
+                        drop-shadow(1px 1px 0 #fff)
+                        drop-shadow(-1px -1px 0 #fff)
+                        drop-shadow(-1px 1px 0 #fff)
+                        drop-shadow(1px -1px 0 #fff)
```

Comment:
This is drop shadow magic to create a somewhat smooth edge around the logo. We can tweak this if you like